wind_cable_poses:
  p1 : [-0.09036419344607977, -0.03417343577152354, 0.0020629677176475514, 3.1411876609741936, 0.00028628981718371605, 1.483329061848989]
  p2 : [-0.08119291544505247, 0.06994282551042819, -0.04830185323953629, 3.141148903300352, 0.0003600593793785638, 1.4832662112472628]
  p3 : [0.009163042684269262, 0.06203770349208948, 0.05566373348236084, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p4 : [0.04079199848197479, 0.07934692517149466, 0.03566373348236085, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p5 : [0.009163042684269262, 0.06203770349208948, 0.05566373348236084, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p6 : [0.0016123010161911422, -0.02364054468440857, 0.05454516291618347, 3.133379731104098, 0.0002858136536033637, 3.077030038757167]
  p7 : [-0.07702920015868986, -0.151267320012053, 0.07411965608596802, 3.133379731104098, 0.0002858136536033637, 3.077030038757167]
  p8 : [-0.024311655088615436, -0.2814126313296167, 0.05461592853069305, 3.13396163180228, 0.0002940788156583138, 3.0773133990971506]
  p9 : [-0.08815430540413127, -0.27842785484117366, -0.025663272738456727, 3.133991991838707, 0.0004650538226599839, 3.0773561428326466]
  p10 : [-0.1253845608979553, -0.08752171019383814, -0.025737845599651338, -3.1225997876436753, -0.0015598575330575332, 1.5117407213000118]
wind_cable_poses2:
  p1 : [-0.09036419344607977, -0.03417343577152354, 0.0020629677176475514, 3.1411876609741936, 0.00028628981718371605, 1.483329061848989]
  p2 : [-0.08119291544505247, 0.06994282551042819, -0.04830185323953629, 3.141148903300352, 0.0003600593793785638, 1.4832662112472628]
  p3 : [0.009163042684269262, 0.06203770349208948, 0.05566373348236084, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p4 : [0.02541329929686037, 0.0756732878222508, 0.040663733482360825, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p5 : [0.009163042684269262, 0.06203770349208948, 0.05566373348236084, 3.141032840874013, 8.841694819062864e-05, 1.0625402885526538]
  p6 : [0.0016123010161911422, -0.02364054468440857, 0.05454516291618347, 3.133379731104098, 0.0002858136536033637, 3.077030038757167]
  p7 : [-0.04082401297499938, -0.15443486345145996, 0.07411965608596802, 3.133379731104098, 0.0002858136536033637, 3.077030038757167]
  p8 : [-0.024311655088615436, -0.2814126313296167, 0.05461592853069305, 3.13396163180228, 0.0002940788156583138, 3.0773133990971506]
  p9 : [-0.08815430540413127, -0.27842785484117366, -0.025663272738456727, 3.133991991838707, 0.0004650538226599839, 3.0773561428326466]
  p10 : [-0.1253845608979553, -0.08752171019383814, -0.025737845599651338, -3.1225997876436753, -0.0015598575330575332, 1.5117407213000118]
